packages/compiler/src/emitter-framework/type-emitter.ts

```diff
@@ -666,6 +666,14 @@ export class TypeEmitter<T, TOptions extends object = Record<string, never>> {
     return this.emitter.result.none();
   }
 
+  tupleLiteralValuesContext(tuple: Tuple): Context {
```

Comment:
I think you can remove the duplicate overide in `CodeTypeEmitter` now

Reply:
Removed.